docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -325,18 +331,24 @@ Release
 
 #. Build RPM packages and upload them to ``archive.clusterhq.com``
 
+   .. note:: Skip this step for a documentation release.
```

Comment:
I think that this means that the gsutil "Access" at the top of this document is not necessary too.